Issue #4292 link to community guidelines at the top (#4296)
open-telemetry/opentelemetry.io: content/en/docs/contributing/_index.md

cSpell:ignore: prepopulated spacewhite
---

> This document is related to contributing do documentation.

Reviewer: ```suggestion
The following guide describes how to contribute to OpenTelemetry documentation.
```

If you are looking on how to contribute to the OpenTelemetry project in general see the

> [OpenTelemetry Contributor Guide](https://github.com/open-telemetry/community/blob/main/CONTRIBUTING.md)

Reviewer 2: could we also include a wording to indicate that every language implementation, collector, conventions repo has their own specific contributing guides as well?

Author: Is there also any place that aggregates all contributing guides?

Author: Added the wording

> If you are looking on how to contribute to the OpenTelemetry project in general see the
> [OpenTelemetry Contributor Guide](https://github.com/open-telemetry/community/blob/main/CONTRIBUTING.md)
> , which provides details on the Contributor License Agreement and the Code of
> Conduct.

Reviewer: ```suggestion
For guidance on how to contribute to the OpenTelemetry project in general, see the
[OpenTelemetry Contributor Guide](https://github.com/open-telemetry/community/blob/main/CONTRIBUTING.md)
, which provides details on the Contributor License Agreement and the Code of
Conduct.
```
